Clean the repository

hubertmis opened the pull request:
This project has been moved to new repository. Source files are
removed from this one to avoid confusion. README is updated to point to
the new location.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver?pullRequest=13) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver?pullRequest=13) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver?pullRequest=13) it.</sub>

hubertmis commented:
I've found a few more labels that needed to be updated. Now should be OK.

Commit:
Clean the repository

This project has been moved to new repository. Source files are
removed from this one to avoid confusion. README is updated to point to
the new location.
